install/threesdk/cli.py

```diff
@@ -170,10 +221,14 @@ def main():
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--expert", default=False, action="store_true")
-
+    parser.add_argument("--update", default=False, action="store_true")
```

Comment:
This update seems to be more a command then a flag.
It make more sense to have something like `3sdk update` then `3sdk --udpdate` specially that the commands exists after execution.

Reply:
changed, thanks for ur note :+1: